op-supervisor/supervisor/backend/db/db_test.go

```diff
@@ -63,6 +301,22 @@ func (s *stubLogDB) AddLog(logHash backendTypes.TruncatedHash, block eth.BlockID
 	return nil
 }
 
+type containsResponse struct {
+	contains bool
+	index    entrydb.EntryIdx
+	err      error
+}
+
+// stubbed Contains records the arguments passed to it
+// it returns the response set in the struct, or an empty response
+func (s *stubLogDB) Contains(blockNum uint64, logIdx uint32, loghash backendTypes.TruncatedHash) (bool, entrydb.EntryIdx, error) {
+	if s.containsResponse != (containsResponse{}) {
+		return s.containsResponse.contains, s.containsResponse.index, s.containsResponse.err
+	} else {
+		return false, 0, nil
+	}
```

Comment:
This can be simplified to just return the `containsResponse` attributes always. They'll be the default zero values if the `containsResponse` as a whole was zeroed.